README.md

```diff
@@ -263,6 +265,30 @@ mOktaAuth.refreshAccessToken(new OktaAuth.OktaAuthListener() {
 
 Tokens are securely stored in the private Shared Preferences.
 
+### End session
+
+In order to perform end session within user's current browser and perform logout
+you have to call `endSession()` whenever you ready
+
+```java
+// LoginActivity.java
+
+public class UserInfoActivity extends Activity {
+
+    private void endSession() {
+        Intent completionIntent = new Intent(this, LoginActivity.class);
+        Intent cancelIntent = new Intent(this, UserInfoActivity.class);
+        cancelIntent.setFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP);
+
```

Comment:
In this example (and in the demo app), add `.logout()` as well to destroy any tokens stored locally.

Reply:
Our SDK clears all tokens in case logout have been performed successfully. 
This method called by `OktaManagementActivity`

`void clearSessionData() {
        // discard the authorization and token state, but retain the configuration and
        // dynamic client registration (if applicable), to save from retrieving them again.
        AuthState currentState = mAuthStateManager.getCurrent();
        AuthState clearedState =
                new AuthState(currentState.getAuthorizationServiceConfiguration());
        if (currentState.getLastRegistrationResponse() != null) {
            clearedState.update(currentState.getLastRegistrationResponse());
        }
        mAuthStateManager.replace(clearedState);
    }`

Reply:
IIRC, the AppAuth library should also support token revocation.

In our example, can we encourage developers to not only **clear** their stored tokens, but revoke the existing `accessToken` and `refreshToken`?

Reply:
@jmelberg-okta That is a good catch we have to do that, but the current implementation of AppAuth-android does not have a revocation functionality implemented yet. I think we need to add support of this feature to AppAuth first and only then update our endSession flow. I will add a feature request to AppAuth android as well.